.env.template

```diff
@@ -26,10 +26,10 @@ MONGODB_URI=mongodb://localhost:27017/
 IS_EDUSHARING_DEPLOYMENT=false
 
 # Assets bucket
-S3_ENDPOINT=https://s3.eu-central-3.ionoscloud.com/
-BUCKET_NAME=editor-media-assets-development
-BUCKET_REGION=eu-central-3
-MEDIA_BASE_URL=https://editor.serlo.dev
+S3_ENDPOINT=#https://s3.eu-central-3.ionoscloud.com/
```

Comment:
So I guess the env values still need to be defined, but can now be empty as well? Would be fine for now.

Reply:
did not try, but I think they could also be removed… 
I thought it's nice to have the example values here still.